Dockerfile

```diff
@@ -7,14 +8,15 @@ COPY ./oracle-client-files/instantclient-basic*-linux.*.zip \
      ./oracle-client-files/instantclient-odbc-linux.*.zip \
      /opt/system/vendor/oracle/
 
-ENV LD_LIBRARY_PATH=/opt/oracle/instantclient/:$LD_LIBRARY_PATH \
+ENV LD_LIBRARY_PATH=/opt/oracle/instantclient/ \
```

Comment:
why change this?

Reply:
apparently, it was causing some issues when building: https://issues.redhat.com/browse/THREESCALE-11393?focusedId=25700046&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25700046